src/StoreApi/Schemas/CartSchema.php

```diff
@@ -265,16 +271,22 @@ public function get_item_response( $cart ) {
 		// Get cart errors first so if recalculations are performed, it's reflected in the response.
 		$cart_errors = $this->get_cart_errors( $cart );
 
+		// The core cart class will not include shipping in the cart totals if `show_shipping()` returns false. This can
+		// happen if an address is required, or through the use of hooks. This tracks if shipping has actually been
+		// calculated so we can avoid returning costs and rates prematurely.
+		$has_calculated_shipping = $cart->show_shipping();
```

Comment:
This won't impact how shipping is shown in the checkout right?

Reply:
Not this specifically but the `null` cases will (we show a message saying shipping will update once address info is complete).